src/main.py

```diff
@@ -59,9 +59,12 @@ def send_mail(
         )
         msg.attach(part)
 
-    smtp = smtplib.SMTP(server, port)
-    if use_tls:
-        smtp.starttls()
+    if smtp_port == 465:
```

Comment:
Should this be `port`? Can you please test if this still works locally for you
```suggestion
    if port == 465:
```